File: bj11399.py
#greedy 알고리즘 공부 - #11399 ATM

import sys
input = sys.stdin.readline

n = int(input())
times = list(map(int, input().split()))
sum = 0

times.sort()
for i in range(n+1):
    for j in range(i):
        sum += times[j]
print(sum)

File: bj11399.c
#include <stdio.h>

int main(void){
    int n;
    scanf("%d", &n);
    int times[n];

    for(int i = 0; i < n; i++){
        scanf("%d", &times[i]);
    }
    for(int i = 0; i < n; i++){
        printf("%d", times[i]);
    }

    return 0;
}